docs/guide/advanced/transitions.md

```diff
@@ -1 +1,53 @@
 # Transitions
+
+In general, you may want to test the resulting DOM after a transition, and this is why Vue Test Utils mocks `<transition>` and `<transition-group>` by default.
+
+Following is a simple component that toggles a content wrapped in a fading transition:
+
+```vue
+<template>
+  <button @click="show = !show">Toggle</button>
+
+  <transition name="fade">
+    <p v-if="show">hello</p>
+  </transition>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      show: false
+    }
+  }
+}
+</script>
+
+<style lang="css">
+.fade-enter-active, .fade-leave-active {
+  transition: opacity 0.5s ease;
+}
+
+.fade-enter-from, .fade-leave-to {
+  opacity: 0;
+}
+</style>
+```
+
+Since Vue Test Utils stubs built-in transitions, you could test the component above as you'd test any other component:
```

Comment:
could -> can (but I'm not a native English speaker, so you probably know better ^^)

Reply:
I'm not native either, so I'll go ahead and change it because "can" sounds good to me, too 😂